modules/installation-special-config-storage.adoc

```diff
@@ -18,25 +35,46 @@ This feature:
 * Requires no user intervention for providing passphrases
 * Uses AES-256-XTS encryption, or AES-256-CBC if FIPS mode is enabled
 
-There are two different supported encryption modes:
+[id="installation-special-config-encryption-threshold_{context}"]
+=== Configuring an encryption threshold
+
+In {product-title}, you can specify a requirement for more than one Tang server. You can also configure the TPM v2 and Tang encryption modes simultaneously, so that the boot disk data can be decrypted only if the TPM secure cryptoprocessor is present and the Tang servers can be accessed over a secure network. 
 
-* TPM v2: This is the preferred mode. TPM v2 stores passphrases in a secure cryptoprocessor.
-* Tang: To use Tang to encrypt your cluster, you need to use a Tang server. Clevis implements decryption on the client side.
+You can use the `threshold:` attribute in your Butane configuration to define the minimum number of TPM v2 and Tang encryption conditions that must be met for decryption to occur. The threshold is met when the stated value is reached through any combination of the declared conditions. For example, the `threshold:` value of `2` in the following configuration can be reached by accessing the two Tang servers, or by accessing the TPM secure cryptoprocessor and one of the Tang servers:
+
+.Example Butane configuration
+
+[source,yaml]
+----
+boot_device:
+  layout: x86_64 
+  luks: 
+    tpm2: true <1>
+    tang: <2>
+      - url: http://tang1.example.com:7500
+        thumbprint: jwGN5tRFK-kF6pIX89ssF3khxxX
+      - url: http://tang2.example.com:7500
+        thumbprint: VCJsvZFjBSIHSldw78rOrq7h2ZF
+    threshold: 2 <3>
+----
+<1> Include this field if you want to use a Trusted Platform Module (TPM) to encrypt the root file system.
+<2> Include this section if you want to use one or more Tang servers.
+<3> Specify the minimum number of TPM v2 and Tang encryption conditions that must be met for decryption to occur. The default value is `1`. See the _Configuring an encryption threshold_ section for more information on this topic.
 
 [IMPORTANT]
 ====
-The use of Tang encryption mode to encrypt your disks is only supported for bare metal and vSphere installations on user-provisioned infrastructure.
+The default `threshold:` value is `1`. If you include multiple encryption conditions in your configuration but do not specify a threshold, decryption can occur if any of the conditions are met.
```

Comment:
`threshold`, without the colon, seems more conventional?

Reply:
Thank you for highlighting this. I have subsequently resolved this throughout the PR.